packages/react-art/src/__tests__/ReactART-test.js

```diff
@@ -24,13 +24,8 @@ import Wedge from 'react-art/Wedge';
 
 // Isolate DOM renderer.
 jest.resetModules();
-
 const ReactDOMClient = require('react-dom/client');
-const act = require('internal-test-utils').act;
-
-// Isolate test renderer.
```

Comment:
Should we move that comment to the new position of `require('react-test-renderer')`?